src/reactviews/pages/UserSurvey/userSurveyPage.tsx

```diff
@@ -0,0 +1,303 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import {
+    Button,
+    Divider,
+    Field,
+    makeStyles,
+    Radio,
+    RadioGroup,
+    Text,
+    Textarea,
+} from "@fluentui/react-components";
+import { useContext, useEffect, useState } from "react";
+import { UserSurveyContext } from "./userSurveryStateProvider";
+import { locConstants } from "../../common/locConstants";
+import { Answer, Question } from "../../../sharedInterfaces/userSurvey";
+
+const useStyles = makeStyles({
+    root: {
+        display: "flex",
+        flexDirection: "column",
+        width: "500px",
+        maxWidth: "100%",
+        "> *": {
+            marginBottom: "15px",
+        },
+        padding: "10px",
+    },
+    title: {
+        marginBottom: "30px",
+    },
+    buttonsContainer: {
+        display: "flex",
+        "> *": {
+            marginRight: "10px",
+        },
+    },
+});
+
+export const UserSurveyPage = () => {
+    const classes = useStyles();
+    const userSurveryProvider = useContext(UserSurveyContext);
+    const [isSubmitDisabled, setIsSubmitDisabled] = useState(true);
+    const [userAnswers, setUserAnswers] = useState<Answer[]>([]);
+
+    useEffect(() => {
+        function loadQuestions() {
```

Comment:
what's the advantage of having `loadQuestions()` as a small function that's only called once locally?

Reply:
I just prefer organizing my useEffect code in small functions. It is mainly useful in making only certain parts of useEffect async without making the entire useEffect async.